PINCache/PINDiskCache.m

```diff
@@ -842,11 +840,8 @@ - (void)setObject:(id <NSCoding>)object forKey:(NSString *)key fileURL:(NSURL **
         PINDiskCacheError(writeError);
         
         if (written) {
-            //Attempting to do this asynchronously seems to result in *worse* performance.
-            [self _locked_setFileModificationDate:now forURL:fileURL];
-            
             NSError *error = nil;
-            NSDictionary *values = [fileURL resourceValuesForKeys:@[ NSURLTotalFileAllocatedSizeKey ] error:&error];
+            NSDictionary *values = [fileURL resourceValuesForKeys:@[ NSURLContentModificationDateKey, NSURLTotalFileAllocatedSizeKey ] error:&error];
```

Comment:
@garrettmoon Great change!  Curious though, what is the reason to request this value?  Couldn't we just use -now, if the function is to update an LRU data structure?